### PR TITLE
Update module

### PR DESCRIPTION
**Pull request by fmdl**
This update add :
- multi account import
- use IBAN in 'account_number' (for France Only)

### CODE REVIEW

**Comment by alexis-via** [comment]
I see that you developped a method _RIBwithoutkey2IBAN(). FYI, for the same need, I use the module "account_bank_statement_import_usability" available in https://github.com/akretion/odoo-usability/tree/8.0/account_bank_statement_import_usability which solves the same problem (match IBAN account number in Odoo for a file which doesn't contain the IBAN) but for all file formats (I have the same issue with OFX for French banks).

Please leave a comment on this PR once you have signed the OCA CLA.

**Comment by gurneyalex** [comment]
travis build of 8.0 was recently greenified. can you rebase?

**Comment by fmdl** [comment]
Hi, I'am new on github, I don't understand how is work, if a make a mistake please be cool.

@gurneyalex what is "rebase" ?

Have a good day.

**Comment by legalsylvain** [comment]
you have to run locally

```
git pull --rebase "OCA_remote_name" 8.0
git push --force "IguanaYachts_remote_name" patch-1
```

File: account_bank_statement_import_fr_cfonb/account_bank_statement_import_fr_cfonb.py [hunk 1]
```diff
@@ -50,6 +52,28 @@ def _parse_cfonb_amount(self, amount_str, nb_of_dec):
         elif amount_str[-1] in credit_trans:
             amount_num = float(amount_str[:-1] + credit_trans[amount_str[-1]])
         return amount_num
+        
+        
+    def _RIBwithoutkey2IBAN(self,banque, guichet, compte):
+        # http://fr.wikipedia.org/wiki/Clé_RIB#V.C3.A9rifier_un_RIB_avec_une_formule_Excel
+        # calcul key rib :
+        lettres = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```
Comment (legalsylvain) [inline account_bank_statement_import_fr_cfonb/account_bank_statement_import_fr_cfonb.py:60]: could you change vars name, setting english words ?

**Pull request closed without merging**